packages/react-search-ui/src/types/Result.js

```diff
@@ -2,5 +2,15 @@ import PropTypes from "prop-types";
 
 import FieldValueWrapper from "./FieldValueWrapper";
 
-// An object where keys are the field names, and values are field values
-export default PropTypes.objectOf(FieldValueWrapper);
+// Typically an object where keys are the field names, and values are field values.
+// Also could be literally any other arbitrary value depending on the particular Search API response.
+// Default views in Search UI know what to do with FieldValueWrapper values, but not arbitrary values, so it
+// is typically better to work with FieldValueWrapper values. So we encourage FieldValueWrapper, but we accept
```

Comment:
Same copy comment as above here

```suggestion
// is usually better to work with FieldValueWrapper values. We encourage FieldValueWrapper, but we accept
```